tests/Unit/PointwiseFunctions/GeneralRelativity/Test_ProjectionOperators.cpp

```diff
@@ -0,0 +1,242 @@
+  auto spec_spatial_projection_IJ = local_spatial_projection_IJ;
+  {
+    const std::array<double, 9> spec_vals = {
+        {40., 42., 42., 42., 42., 42., 42., 42., 46.}};
+    for (size_t j = 0; j < SpatialDim; ++j) {
+      for (size_t k = j; k < SpatialDim; ++k) {
+        spec_spatial_projection_IJ.get(j, k) =
+            gsl::at(spec_vals, j * SpatialDim + k);
+      }
+    }
+  }
+
+  // Compare values returned to those from SpEC
+  CHECK_ITERABLE_APPROX(local_spatial_projection_IJ,
+                        spec_spatial_projection_IJ);
+
+  // 2. Projection ij
+  auto local_spatial_metric =
+      make_with_value<tnsr::ii<DataVector, SpatialDim, Frame::Inertial>>(
+          inertial_coords, 0.);
+  auto local_unit_interface_normal_one_form =
+      make_with_value<tnsr::i<DataVector, SpatialDim, Frame::Inertial>>(
+          inertial_coords, 0.);
+  auto local_spatial_projection_ij =
+      make_with_value<tnsr::ii<DataVector, SpatialDim, Frame::Inertial>>(
+          inertial_coords, 0.);
+
+  // Setting inverse_spatial_metric to compare with values from SpEC
+  for (size_t i = 0; i < SpatialDim; ++i) {
+    local_spatial_metric.get(0, i) = 263.;
+    local_spatial_metric.get(1, i) = 269.;
+    local_spatial_metric.get(2, i) = 271.;
+  }
+  // Setting unit_interface_normal_vector to compare with values from SpEC
+  get<0>(local_unit_interface_normal_one_form) = -1.;
+  get<1>(local_unit_interface_normal_one_form) = 1.;
+  get<2>(local_unit_interface_normal_one_form) = 1.;
+
+  // Call tested function
+  gr::transverse_projection_operator(
+      make_not_null(&local_spatial_projection_ij), local_spatial_metric,
+      local_unit_interface_normal_one_form);
+
+  // Initialize with values from SpEC
+  auto spec_spatial_projection_ij = local_spatial_projection_ij;
+  {
+    const std::array<double, 9> spec_vals = {
+        {262., 264., 264., 264., 268., 268., 264., 268., 270.}};
+    for (size_t j = 0; j < SpatialDim; ++j) {
+      for (size_t k = j; k < SpatialDim; ++k) {
+        spec_spatial_projection_ij.get(j, k) =
+            gsl::at(spec_vals, j * SpatialDim + k);
+      }
+    }
+  }
+
+  // Compare values returned to those from SpEC
+  CHECK_ITERABLE_APPROX(local_spatial_projection_ij,
+                        spec_spatial_projection_ij);
+
+  // 3. Projection Ij
+  auto local_spatial_projection_Ij =
+      make_with_value<tnsr::Ij<DataVector, SpatialDim, Frame::Inertial>>(
+          inertial_coords, 0.);
+
+  // Call tested function
+  gr::transverse_projection_operator(
+      make_not_null(&local_spatial_projection_Ij),
+      local_unit_interface_normal_vector, local_unit_interface_normal_one_form);
+
+  // Initialize with values from SpEC
+  auto spec_spatial_projection_Ij = local_spatial_projection_Ij;
+  {
+    const std::array<double, 9> spec_vals = {
+        {0., 1., 1., 1., 0., -1., 1., -1., 0.}};
+    for (size_t j = 0; j < SpatialDim; ++j) {
+      for (size_t k = j; k < SpatialDim; ++k) {
```

Comment:
This only iterates over the upper-triangular matrix elements, and because it is initialized with the expected object before the assignment, the missing assignments aren't obvious.
I suggest initializing the `spec_spatial_projection...` to the default constructed type (or `make_with_values` if you prefer), so that it's more obvious that the test and expected are independently constructed.

Reply:
makes sense, changed to initializing with `make_with_value` everywhere.